### ISSUE TITLE
Please add paths to documentation for specific messages

### ISSUE THREAD

**Issue by ThingsOfficer** [ttnmapper/documentation#7]
Hi JP & other contributors!

It just took me quite a while right now to adapt the TTN Mapper integration correctly as webhook. 
I had to work through old issues and current source code to find out how to configure the paths for the messages properly. Please add it to documentation to help the next with this issue. 

Best regards!

<img width="278" alt="Bildschirmfoto 2021-12-23 um 13 39 01" src="https://user-images.githubusercontent.com/61187940/147242269-ccef7518-4c0b-404d-8f0c-4728ce7196f7.png">

**Comment by jpmeijers** [comment]
I do not see why this should be documented. The documentation tells the user to use the webhook template that is available in the TTS Console. The template sets up all the required paths and headers for the integration to work. Manual editing of the webhook is neither required, nor should it be done. Manual editing will most likely result in the webhook to not work, as TTN Mapper is very strict about how and from where data is ingested.

Can you please explain what your original issue was, and why you ended up manually modifying the webhook? Most likely there is another reason for things to not work, which we should solve differently.

**Comment by ThingsOfficer** [comment]
Ok, yes, so indeed there was a small failure on my side first, which led me to inspect the webhook settings. There I saw I had only the upload-messages path configured and found out by different sources that solved-location and join-accept was also defined. Unsure about where I was wrong I suspected the failure to be here. Anyway I thought it would still be helping to see how the webhook should look like after proper setup. Maybe my template integration was older and only the upload-messages was integrated by then?
